master.js

```diff
@@ -366,7 +366,16 @@ class AlAzureMaster {
             var status;
             if (errStatus) {
                 master._azureContext.log.warn('Health check failed with',  errStatus.details);
```

Comment:
if `errStatus` is string then this warning should be re-formatted as well

Reply:
Good spot.